### PR TITLE
Fix: Update HE Cache  bias command line input

### PR DESCRIPTION
**Pull request by anandaravuri**
-Change HE Cache  bias command line to –bias host or device 
-Change scenario  setup message  to “Pretest scenario started..”  and  actual test message Test started ......"
-Change Fails to allocate 2MB huge page: Invalid argument to “Failed to allocate 2MB huge page: Invalid argument”
-Fix  output message “number of nodes on system”
-Set Stride value only if user passes command line  --stride value.

### CODE REVIEW

File: samples/cxl_host_exerciser/cxl_he_cache_cmd.h [hunk 1]
```diff
@@ -50,7 +50,8 @@ class he_cache_cmd : public he_cmd {
 public:
   he_cache_cmd()
       : he_continuousmode_(false), he_contmodetime_(0), he_linerep_count_(0),
-        he_stide_(0), he_test_(0), he_test_all_(false), he_dev_instance_(0) {}
+        he_stide_(0), he_test_(0), he_test_all_(false), he_dev_instance_(0),
```
Comment (tswhison) [inline samples/cxl_host_exerciser/cxl_he_cache_cmd.h:53]:
```suggestion
        he_stride_(0), he_test_(0), he_test_all_(false), he_dev_instance_(0),
```

File: samples/cxl_host_exerciser/cxl_he_cache_cmd.h [hunk 2]
```diff
@@ -50,7 +50,8 @@ class he_cache_cmd : public he_cmd {
...
+        he_stide_cmd_(false) {}
```
Comment (tswhison) [inline samples/cxl_host_exerciser/cxl_he_cache_cmd.h:54]:
```suggestion
        he_stride_cmd_(false) {}
```

Comment (anandaravuri) [reply samples/cxl_host_exerciser/cxl_he_cache_cmd.h]: Fixed

File: samples/cxl_host_exerciser/cxl_he_cache_cmd.h (hunk 1 above)
Comment (anandaravuri) [reply samples/cxl_host_exerciser/cxl_he_cache_cmd.h]: Fixed

File: samples/cxl_host_exerciser/cxl_he_cmd.h [hunk 3]
```diff
@@ -239,29 +242,38 @@ class he_cmd : public test_command {
...
-          he_ctl_.bias_support = 0x0;
-          break;
+    // Target memory HOST set BIAS host
+    if (he_target_ == HE_TARGET_HOST) {
+        he_ctl_.bias_support = HOSTMEM_BIAS;
+        // Target memory FPGA set BIAS host/device
+        if (he_bias_ == HOSTMEM_BIAS) {
+            he_ctl_.bias_support = HOSTMEM_BIAS;
+        } else {
+            cerr << "Invalid configuration:Target memory host and bias device"
```
Comment (KrishnaSimmadhari) [inline samples/cxl_host_exerciser/cxl_he_cmd.h:254]: Rewrite this error: "Wrong BIAS mode for specified target memory type"

File: samples/cxl_host_exerciser/cxl_he_cache_cmd.h [hunk 4]
```diff
@@ -94,9 +95,9 @@ class he_cache_cmd : public he_cmd {
         ->default_val("host");
 
     app->add_option("--bias", he_bias_,
-        "host exerciser run on hostmem or fpgamem")
+        "host exerciser run on host or device")
```
Comment (KrishnaSimmadhari) [inline samples/cxl_host_exerciser/cxl_he_cache_cmd.h:98]: Change the help to "CXL IP Memory access Bias mode"

**Review by KrishnaSimmadhari — CHANGES_REQUESTED** [review]
Please do the necessary changes and checkin.

File: samples/cxl_host_exerciser/cxl_he_cmd.h (hunk 3 above)
Comment (anandaravuri) [reply samples/cxl_host_exerciser/cxl_he_cmd.h]: Fixed

File: samples/cxl_host_exerciser/cxl_he_cache_cmd.h (hunk 4 above)
Comment (anandaravuri) [reply samples/cxl_host_exerciser/cxl_he_cache_cmd.h]: Fixed

**Review by tswhison — APPROVED** [review]

**Review by KrishnaSimmadhari — APPROVED** [review]

**Pull request merged**